docs: Add Email Notifications info (#9930)
immich-app/immich: docs/docs/administration/email-notification.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# Email Notifications

> Immich supports the option to send notifications via email, this option is supported for:

Reviewer: ```suggestion
Immich supports the option to send notifications via email. Currently, this is supported for:
```

> - Creating a new user .
> - A user is added to a shared album (send email to the newly shared user).
> - A shared album is added with new assets (send an email to all shared users beside the one that is adding assets).

Reviewer: ```suggestion
- Creating a new user
- Notifying a user when they get added to a shared album
- Informing other users about the addition of new assets to a shared album
```

> The easiest way to set up a mail server for notifications is through Google, you may want to avoid using Gmail for privacy reasons or as part of your degoogle plan, you can use any SMTP server that can support login using name and password.

Reviewer 2: Maybe this will be better in the Guide section. In this section we just describe that is the Email notification feature, how can we set it up, what are the parameter...etc

Author: Maybe remove this section?

Reviewer: I don't think we should remove this section completely, as this is a scenario many (if not most) users will run into. I agree with Alex in that we should move this section to a guide though

Author: So what sections would we like to leave on this page?

## Create an app password

> First, it is recommended to create a dedicated Google account for your Immich instance, and not to mix it with your main Google account.

Reviewer 2: Is this your opinion? We should be subjective in writing documentation

Reviewer: We should *not* be subjective :D

Author: I put it in a :::tip:::, if you think it should be removed, tell me.

Under Email, enter the following details for connection with Gmail servers:

| Setting                   | Value                                                                          |
| :------------------------ | :----------------------------------------------------------------------------- |
| Host                      | `smtp.gmail.com`                                                               |
| Port                      | 587 (For TLS) 465 (For SSL)                                                    |
| Username                  | Your Google account address                                                    |
| Password                  | Your created password from Google app passwords page                           |
| Ignore certificate errors | Disable                                                                        |
| From address              | Sender email address, for example: "Immich Photo Server`<noreply@immich.app>`" |

</TabItem>
<TabItem value="Mobile" label="Mobile">

Immich does not support this setting through the mobile app at this time.

</TabItem>
</Tabs>

---

> :::danger plain password

Reviewer: ```suggestion
:::danger Plain password
```

> :::danger plain password

Reviewer 2: This information is not related to the email notification feature. It can be included in the user creation session. Try to keep the documentation short and to the point

Author: Agreed, I fixed it.

> Email messages sent by a user contain the login password you set as plain text, it is recommended to choose a password that is not too easy but not too personal. In any case, unless otherwise specified, the user will be asked to enter a new password when logging in.

Reviewer: ```suggestion
Invitation emails contain the initial password you set as plain text. It is recommended to choose a password that is neither too easy nor too personal. In any case, unless otherwise specified, the user will be asked to enter a new password when logging in.
```
